### ISSUE TITLE
Mojaloop GP - pre cleanup and post clean up scripts failing when tested on mojaloop helm v15 release

### ISSUE THREAD

**Issue by sri-miriyala** [mojaloop/project#3165]
**Summary**:
Some of the settlement related tests in pre GP cleanup and post GP cleanup, especially around "create settlement" are failing when tested against  mojaloop helm v15 release in moja2 env.

**Severity**:
(Low, Medium, High)

**Priority**:
(Critical, Medium, Low)

**Expected Behavior**

Pre and post cleanup scripts to be successful without failures on mojaloop helm release v15.0.0

**Steps to Reproduce**
  1. On a deployment with v15.0.0 RC / dev release, run tests from here: https://github.com/mojaloop/testing-toolkit-test-cases/tree/v14.1.0
  2. Failures on pre GP cleanup and post GP cleanup can be seen.
  
Tasks:
- [x] Create separate collections for pre/post clean-up so that it these collections can be used selectively as needed
- [x] Validate the fix by running multiple times
- [x] Remove the FundsIn collection from GP
- [x] Validate settlements position changes before GP, after GP, and after cleanup
- [x] Validate the interchange fee positions
- [x] Add steps to run clean up collections to documentation in testing-toolkit-test-cases and Helm release notes (testing notes section) 
- [x] Add cleanup collection to the helm tests [@mdebarros]

**Specifications**
  - Component (if known):
  - Version: helm v15.0.0 RC (or dev release)
  - Platform: Mojaloop
  - Subsystem:
  - Type of testing: functional
  - Bug found/raised by: @mdebarros 

**Notes**:

- Severity when opened:
- Priority when opened:

**Tasks\Solution**

**Comment by sri-miriyala** [comment]
<img width="1269" alt="Story3165" src="https://user-images.githubusercontent.com/38143240/226632597-6e5ad31c-b809-4e6e-ae53-2e34b6a711dd.png">

The above scree shot is showing the position values of test FSPs in different stages.
Those stages are 
1. Before running GoldenPath collection 
2. After Running GP 
3. After running cleanup collection
The red arrows showing the position values in these 3 different stages. The goal is to reset positions through the clean up collection, so that we will not have issues of failing tests due to insufficient funds on the payer fsp when transfers run during tests. The above screenshot is showing the position values are reset after the change in positions due to GP test run.
Also the FundsIn into different test FSPs have been removed as that is not needed anymore.
As part of this task, I have noticed the noresponsepayeefsp is showing up a position increase which should not be the case as this fsp is specifically used in transfers error cases, where the transfers fail and no change in positions. Below issue is created to look deeper into the issue:
